product-detail-page/product-page.css

```diff
@@ -0,0 +1,111 @@
+@import url(/product-list/product-list.css);
+
+.product-main-container{
+    margin: 9rem 0 1rem 0;
+}
+.product-container{
+    align-items: flex-start;
+    gap: 2rem;
+}
+
+.product-img-container{
+    min-width: 35rem;
+    width: 35rem;
+}
+.product-img-container > img{
+    height: auto;
+    object-fit: cover;
+    max-inline-size: 100%;
+}
+
+.product-details-container{
+    padding-inline: 1rem;
+}
+
+.product-details-container > *{
+    margin: 1rem 0;
+}
+
+.product-description-box{
+    margin: 1rem 0;
+}
+.product-description{
```

Comment:
Try to use relative units. It will be more helpful while adding responsiveness.
Else LGTM!!